easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023b.eb

```
@@ -150,7 +150,8 @@ exts_list = [
         'source_urls': ['https://github.com/tensorflow/tensorflow/archive/'],
         'test_script': 'TensorFlow-2.x_mnist-test.py',
         'test_tag_filters_cpu': '-gpu,-tpu,-no_cuda_on_cpu_tap,-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only',
-        'test_tag_filters_gpu': 'gpu,-no_gpu,-nogpu,-gpu_cupti,-no_cuda11,-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only',
+        'test_tag_filters_gpu': ('gpu,-no_gpu,-nogpu,-gpu_cupti,-no_cuda11,'
+        '-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only'),
```

Sorry, I think this needs to be aligned with the ' on the line above.
```suggestion
                                 '-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only'),
```